contents/docs/surveys/implementing-custom-surveys.mdx

```diff
@@ -83,4 +83,50 @@ posthog.capture("survey sent", {
     $survey_id: {survey.id} // required
     $survey_response: {survey_response} // required
 })
-```
\ No newline at end of file
+```
+
+### Question types
+
+If you want to make the most of PostHog's automatic survey analysis, you'll need to capture survey responses in these formats:
+
+#### Open text and single choice questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: {response} // value of the text input or selected choice
+})
+```
+
+#### Multiple choice questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: ["response_1", 8, "response_2"] // array of values of the selected choices
+})
+```
+
+#### Rating questions
+
+```js-web
+posthog.capture("survey sent", {
+    $survey_id: {survey.id} // required
+    $survey_response: {rating} // rating is a number
+})
+```
+
```

Comment:
```suggestion
### Question types

If you want to make the most of PostHog's automatic survey analysis, you'll need to capture survey responses in these formats:

#### Open text and single choice questions

```js-web
posthog.capture("survey sent", {
    $survey_id: {survey.id} // required
    $survey_response: {response} // value of the text input or selected choice
})
```

#### Multiple choice questions

```js-web
posthog.capture("survey sent", {
    $survey_id: {survey.id} // required
    $survey_response: ["response_1", 8, "response_2"] // array of values of the selected choices
})
```

#### Rating questions

```js-web
posthog.capture("survey sent", {
    $survey_id: {survey.id} // required
    $survey_response: {rating} // rating is a number
})
```
```